components/form/KeyValue.vue

```diff
@@ -348,10 +355,33 @@ export default {
             out[key] = value;
           }
         }
+        this.$emit('input', out);
       }
-      this.$emit('input', out);
-    }
-  },
+    },
+
+    displayProps(value) {
+      const binary = typeof value === 'string' && !asciiLike(value);
+      const isFormatted = typeof value === 'string' ? value.includes('-----BEGIN') : false;
```

Comment:
Looking for `-----BEGIN` is not a good assumption... lots of other kinds of values (YAML, JSON, TOML, etc) have newlines in them and they should be shown with them.

Using `<pre>` also gets you back into horizontal scrolling.

```javascript
displayValue = escapeHtml(value).replace(/(\r\n|\r|\n)/g, '<br/>\n');
```

```html
<span v-html="row.displayValue"/>
```